src/assets/i18n/en.json

```diff
@@ -1,3 +1,12 @@
 {
-	"test": "Test text"
+	"Recipes": {
```

Comment:
we use camelCase for translations in our projects so it would be good if you can adjust it here also